cmd/init.go

```diff
@@ -53,6 +58,7 @@ var InitCmd = &cobra.Command{
 
 func init() {
 	InitCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Deploy Dapr to a Kubernetes cluster")
+	InitCmd.Flags().BoolVarP(&slimMode, "slim", "s", false, "Initialize dapr in self-hosted mode without docker.")
```

Comment:
Rephrase "without docker" to "without Redis and Actor Placement containers" or something similar